README.md

```diff
@@ -37,6 +37,10 @@ This application will act as the controller for the warm pool and ensure that th
   - Resource Group
   - Subnet with internet access
   - KeyVault for GitHub App Credential
+  - Service Principal Roles/Permissions
+    - `Microsoft.AppConfiguration/locations/deletedConfigurationStores/read` (Needed until closer of [issue #19605 in hashicorp/terraform-provider-azurerm](https://github.com/hashicorp/terraform-provider-azurerm/issues/19605))
```

Comment:
is `closer` a typo?

Reply:
Yep, was meant to be `closure`! Good catch, thanks!